### PR TITLE
docs: CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by popey**
This PR adds a code of conduct document to the repo, as agreed at our recent OSS team catch up.

### CODE REVIEW

**Review by spiffcs — APPROVED** [review]

**Comment by spiffcs** [comment]
Waiting on other members of the project before full merge 😄

**Review by wagoodman — APPROVED** [review]

**Pull request merged**
